src/modules/project/funding/state/fundingStagesAtom.ts

```diff
@@ -0,0 +1,52 @@
+import { atom, useAtom, useSetAtom } from 'jotai'
+import { atomEffect } from 'jotai-effect'
+
+import { findNextFundingStage } from '../../utils/helpers'
```

Comment:
Seems the `findNextFundingStage` is only called here, might as well keep the function locally no?

Reply:
that's a good point, and yes.